src/webportal/src/app/home/home/virtual-cluster-statistics.jsx

```diff
@@ -101,8 +101,8 @@ const vcListColumns = [
     isResizable: true,
     onRender(vc) {
       const { resourcesUsed, resourcesTotal } = vc;
-      const resourceAvailable = vc.resourceAvailable || resourcesTotal;
-      const insufficient = !(resourceAvailable.GPUs === resourcesTotal.GPUs);
+      const resourcesGuaranteed = vc.resourcesGuaranteed || resourcesTotal;
+      const insufficient = !(resourcesGuaranteed.GPUs === resourcesTotal.GPUs);
```

Comment:
Comment and expose resourcesGuaranteed is the (Total - Bad)

Reply:
changed to show "used / total - bad", no guaranteed.